### ISSUE TITLE
wine-lol-autoconf-2.70.patch

### ISSUE THREAD

**Issue by fefemess** [M-Reimer/wine-lol#77]
wine-lol-autoconf-2.70.patch checksum fails
It's a clean install of arch so there's no old version already installed that might be getting mistaken

**Comment by M-Reimer** [comment]
So GitHub did it again... But this time, I still had the old file at hand to check this out:

```diff
--- wine-lol-autoconf-2.70.patch.old	2021-04-07 12:05:07.413090486 +0200
+++ wine-lol-autoconf-2.70.patch	2021-07-08 07:42:01.239020598 +0200
@@ -24,5 +24,5 @@
 
 diff --git a/aclocal.m4 b/aclocal.m4
-index c5edd85cee49..bd8ca6b33e55 100644
+index c5edd85cee4..bd8ca6b33e5 100644
 --- a/aclocal.m4
 +++ b/aclocal.m4

```

So GitHub went from 11 characters for the IDs to 12 characters and now back to 11... This effectively makes GitHub patch export useless for directly using in PKGBUILDs if the result is not stable.

**Comment by M-Reimer** [comment]
Fixed in https://github.com/M-Reimer/wine-lol/commit/7ebb4295dd418728a84aed611eff782672ffcab0

**Comment by fefemess** [comment]
> Fixed in [7ebb429](https://github.com/M-Reimer/wine-lol/commit/7ebb4295dd418728a84aed611eff782672ffcab0)

Cheers, works again. Thanks
